src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/velocity/mode/VelocityWatchdog.kt

```diff
@@ -75,12 +54,7 @@ internal object VelocityWatchdog : Choice("Watchdog") {
     @Suppress("unused")
     private val gameHandler = repeatable {
         if (player.isOnGround) {
-            blinkUntilGround = false
-        }
-
-        // Check if there is enemy nearby
-        if (hits > 0 && world.findEnemy(range = 0f..4f) == null) {
-            hits = 0
+            absorbedVelocity = false;
```

Comment:
remove ;

Reply:
this is a really annoying habit, especially that its grey in intellij so i can't see :sob: